elide-core/src/main/java/com/yahoo/elide/core/filter/InMemoryFilterOperation.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.core.filter;
```

Comment:
It's not restricted to be used in InMemoryDataStore only. Every datastore could have use cases when they need to do some in-memory filtering in addition to pushing the filters to the backend data store